patterns/full-width-cover-banner.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Title: Full-width Cover Banner
+ * Slug: twentytwentyfive/full-width-cover-banner
+ * Categories: banner, call-to-action, media, featured
```

Comment:
We should add the "Pages" category to this one too since it's a coming soon landing page that someone would want to set as a standalone page.